templates/desktop/organisations.html

```diff
@@ -0,0 +1,192 @@
+          </div>
+        </li>
+      </ul>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip--security is-bordered">
+  <div class="row">
+    <div class="col-6">
+      <h2>Ubuntu is at the forefront of safety and reliability</h2>
+      <p>Your Ubuntu software is built with unrivalled security in mind — and is secure from the moment you install it, and will remain so because Canonical’s team of security experts will ensure that security updates are always available on Ubuntu first.</p>
+      <p><a href="/security">Learn more about the security of Ubuntu&nbsp;&rsaquo;</a></p>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip--light is-bordered">
+  <div class="row">
+    <div class="col-6 u-hide--small">
+      <img src="{{ ASSET_SERVER_URL }}c6504e94-Dell_XPS_Laptop_Front-Desktop.png?w=640" alt="">
+    </div>
+    <div class="col-6">
+      <h2>Pre-installed on certified hardware</h2>
+      <p>Ubuntu runs on the world’s best hardware from partners such as Dell, HP and Lenovo. And with nearly 90% of PCs shipped by the major PC companies already certified to work with Ubuntu, it is easy to find hardware that will work for your business.</p>
+      <p><a class="p-link--external" href="https://certification.ubuntu.com">Learn more about Ubuntu certified hardware</a></p>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip is-bordered">
+  <div class="row u-vertically-center">
+    <div class="col-6">
+      <h2>The developer&rsquo;s favourite</h2>
+      <p>Ubuntu is used by thousands of development teams around the world because of its versatility, reliability, constantly updated features, and extensive developer libraries.</p>
+      <p>If you’re managing developers, Ubuntu is the best way to increase your team’s productivity and guarantee a smooth transition from development all the way to production.</p>
+      <p><a href="/cloud">Learn more about Ubuntu’s cloud offering&nbsp;&rsaquo;</a></p>
+    </div>
+    <div class="col-6 u-hide--small">
+      <img src="{{ ASSET_SERVER_URL }}a64aed2d-most-popular-opensource-IOS.png" alt="">
+    </div>
+  </div>
+</section>
+
+<section class="p-strip is-bordered">
+  <div class="row">
+    <div class="col-6">
+      <h2>Support and management tools</h2>
+      <p>Ubuntu Advantage is the professional support package from the experts at Canonical. Get 24-hour phone and web support every day with access to engineers with first-hand experience of your issues. It also includes Landscape, the Ubuntu systems management tool, for monitoring, managing, patching and compliance reporting on all your Ubuntu desktops.</p>
+      <p><a href="/support">Learn about Ubuntu Advantage&nbsp;&rsaquo;</a></p>
+    </div>
+    <div class="col-6 u-hide--small">
+      <img src="{{ ASSET_SERVER_URL }}4125e046-landscape_activity.png" alt="">
+    </div>
+  </div>
+</section>
+
+<section class="p-strip--light">
```

Comment:
can you update with this image? https://assets.ubuntu.com/v1/d8c957cc-2018-04-20-release-eof-desktop.png  also, it should be on the left from the design